src/components/layout.js

```diff
@@ -32,6 +32,7 @@ const Layout = ({ children }) => {
       </div>
       <div className='content'>
         {children}
+        <Footer />
```

Comment:
Maybe better way will be move that `<Footer />` below `content` container and set width. Just for make some empty place you can set in `content` some static height and then you can get rid of `align-items: flex-end` in `content` class. (`flex-end` will be undesirable behavior when you pass some data to `Layout` component)